### ISSUE TITLE
Debugger diassembly display not showing proper bank

### ISSUE THREAD

**Issue by mjallison42** [commanderx16/x16-emulator#318]
If debugger hits a break point in a ROM bank, the disassembly display always seems to show ROM bank 00. To test, set a break point in some ROM bank (non-zero) and cause the machine to execute code in that bank. 

* Disassembly code shows proper PC value, but wrong bank, and the code disassembled is not correct. 

This may also be true of executing code in a non-zero RAM bank, but I have not verified this.

**Comment by kktos** [comment]
yep. I think that's because the BP is tested on 16bits address rather than 24bits.
Thanks for your sharp eyes :)
I did fix it on the PR I wrote.
